rust/src/quic/frames.rs

```diff
@@ -284,6 +314,8 @@ fn parse_quic_handshake(msg: TlsMessage) -> Option<Frame> {
                 let mut ja3 = String::with_capacity(256);
                 ja3.push_str(&u16::from(ch.version).to_string());
                 ja3.push(',');
+                let mut ja4 = JA4::new();
```

Comment:
Should the compile-time option prevent this creation ?

Reply:
I tried not to do it that way; introducing an `Option` here would have led to more complicated code throughout the rest of the QUIC handling, which I wanted to avoid.
In case JA4 is going to be disabled the operations on that object are going to be nops.

Reply:
I was not talking about `Option` but `#cfg[ja4]` (or something like that)